image_explorer/image_explorer.py

```diff
@@ -37,7 +37,11 @@
 from parsel import Selector
 from xblock.completable import XBlockCompletionMode
 from xblock.core import XBlock
-from xblock.fragment import Fragment
+try:
+    from web_fragments.fragment import Fragment
+except:
+    # for backwards compatibility with quince and prior releases
+    from xblock.fragment import Fragment
```

Comment:
Do we need this? The Drag and Drop v2 XBlock has been working without such a fallback for over four years: https://github.com/openedx/xblock-drag-and-drop-v2/commit/5334f4dd6375cdffb0dca470d6d4814e9250cfed.